t/testfiles/verb.exp.ltx

```diff
@@ -13,7 +16,7 @@ Test *M \textbackslash{}\{\#!\_<"> e*\forcelinebreak
 Test **M \textbackslash{}\{\#!\_<"> e**\forcelinebreak 
 Test ***M \textbackslash{}\{\#!\_<"> e***\forcelinebreak 
 Test =M \textbackslash{}\{\#!\_<"> e=\forcelinebreak 
-Test <verbatim> </verbatim>
+Test <verbatim> <\Slash{}verbatim>
```

Comment:
Not sure what happened here, but it does not seem harmful

Reply:
It comes out from this sequence `Test <verbatim><verbatim></verbatim> <verbatim></verbatim></verbatim>`
 . Your output looks more correct, but I believe only by chance. Needs further digging an more tests. In the exiting code the characters inside verbatim are just escaped, so the slash is kept as is. Not a big deal, as it's not a special character per se. However, the testfile change points out that the sequence is interpreted differently (because the second escaping is considered empty and the last </verbatim> is left as is.

It would be good if you added more tests with corner cases like this.